scripts/roles.json

```diff
@@ -5061,10 +5528,10 @@
       "aria-relevant",
       "aria-required",
       "aria-roledescription",
-      "aria-valuemax",
-      "aria-valuemin",
       "aria-valuetext",
-      ["aria-valuenow", "0"]
+      ["aria-valuemax", "that there is no maximum value"],
```

Comment:
I don't want to introduce strings that aren't tokens. At least not more than we must. Can you just leave this item without a default?

Reply:
Done